hacks/install-nm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# TODO: Iterate all the nodes
```

Comment:
this will happen only with multi nodes PR, right?

Reply:
Yep, let's fix this there, the idea is to test at jenkins with multiple nodes (We have to make test transparent to the number of nodes).